index.js

```diff
@@ -51,24 +48,25 @@ function decode(bbmsg) {
  * @returns {Buffer} a bendy-butt message encoded with `bencode`
  */
 function encode(msgVal) {
-  const contentSection =
-    typeof msgVal.content === 'string' && msgVal.content.endsWith('.box2')
-      ? bfe.encodeBendyButt(msgVal.content)
-      : [
-          bfe.encodeBendyButt(msgVal.content),
-          bfe.encodeBendyButt(msgVal.contentSignature),
-        ]
-
-  return bencode.encode([
-    [
-      bfe.encodeBendyButt(msgVal.author),
-      bfe.encodeBendyButt(msgVal.sequence),
-      bfe.encodeBendyButt(msgVal.previous),
-      bfe.encodeBendyButt(msgVal.timestamp),
-      contentSection,
-    ],
-    bfe.encodeBendyButt(msgVal.signature),
```

Comment:
It was unnecessary to call `bfe.encodeBendyButt` on each of these, because this function knows how to recursively encode arrays-in-arrays, so we can just `bfe.encodeBendyButt` the top-level array and it'll trickle down.